app/controllers/application_controller.rb

```diff
@@ -27,6 +27,38 @@ class ApplicationController < ActionController::Base
 
   helper_method :current_person
 
+  helper_method def url_for(options)
```

Comment:
These two methods are arcane enough that I think they warrant at least a one-sentence comment explaining that they augment the standard Rails helpers to make URLs on branded domains nicer.